enhancements/network-observability/network-observability.md

```diff
@@ -120,23 +122,34 @@ However, the first release will not analyze why something went wrong.
 
 ### API Extensions
 
-The OVS will be configured to enable NetFlows by using Cluster Network Operator
+The OVS can be configured to enable NetFlows by using Cluster Network Operator
 (networks.operator.openshift.io) to make this change.  This will impact the
 performance of the OVS.
 
+A priviledged eBPF agent can run on each cluster node as an alternative to 
+generate NetFlows. This will impact the performance of nodes with a lower overall 
+consumption than OVS approach.
+
 ### Implementation Details/Notes/Constraints
 
 Here are the limitations and constraints.
 
-1. CNI must be OVN-Kubernetes<br>
-    The network type (CNI) has to be OVN-Kubernetes since configuring OVS to
-export IPFIX data is only supported there.  By default, the CNI is OpenShift
-SDN and the overwhelming majority of customers run OpenShift SDN today.
-Therefore in a brown field scenario, this would require the user to change
-their CNI type before network observability can be enabled.
-    Note that the effort to implement OpenShift SDN to configure OVS is
-doable.  However, new features recently added have only been implemented
-in OVN-Kubernetes.
+1. NetFlows export<br>
+
+    - CNI must be OVN-Kubernetes for OVS NetFlows
+        The network type (CNI) has to be OVN-Kubernetes since configuring OVS to
+    export IPFIX data is only supported there.  By default, the CNI is OpenShift
+    SDN and the overwhelming majority of customers run OpenShift SDN today.
```

Comment:
This is less and less true, ovn-k usage is growing. Maybe remove "overwhelming" here => "the majority of customers" is enough.